continuous-pipe.yml.erb

```diff
@@ -0,0 +1,192 @@
+variables:
+  MYSQL_DATABASE: <%= config.name %>
+  MYSQL_USER: <%= config.name %>
+
+defaults:
+  cluster: ${CLUSTER}
+  environment:
+    name: '"<%= config.name %>" ~ code_reference.branch'
+
+tasks:
+  web_image:
+    build:
+      environment:
+        GITHUB_TOKEN: ${GITHUB_TOKEN}
+
+    services:
+      web:
+        image: quay.io/organisation/<%= config.name %>
+
+  storage:
+    deploy:
+      services:
+        mysql:
+          specification:
+            volumes:
+              - type: persistent
+                name: database-volume
+                capacity: 5Gi
+                storage_class: default
+
+            volume_mounts:
+              - name: database-volume
+                mount_path: /var/lib/mysql
+
+            command:
+              - /usr/local/bin/docker-entrypoint.sh
+              - mysqld
+              - --ignore-db-dir=lost+found
+              - --max_allowed_packet=128M
+
+            ports:
+              - 3306
+
+            resources:
+              requests:
+                cpu: 50m
+                memory: 250Mi
+              limits:
+                cpu: 500m
+                memory: 2Gi
+
+            environment_variables:
+              MYSQL_PASSWORD: ${DATABASE_PASSWORD}
+              MYSQL_ROOT_PASSWORD: ${DATABASE_ROOT_PASSWORD}
+              MYSQL_DATABASE: ${DATABASE_NAME}
+              MYSQL_USER: ${DATABASE_USER}
+
+  reserve_ip:
```

Comment:
This can be removed if setup is done after the deployment of the other services.